charts/renku/templates/secrets.yaml

```diff
@@ -16,10 +16,11 @@ data:
   {{- else }}
   gitlab-sudo-token: {{ randAlphaNum 10 | b64enc | quote }}
   {{- end }}
-  gitlab-password: {{ .Values.global.gitlab.postgresPassword | b64enc | quote }}
+  gitlab-postgres-password: {{ .Values.global.gitlab.postgresPassword | b64enc | quote }}
 {{- end }}
 {{- if .Values.keycloak.enabled }}
-  keycloak-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
+  keycloak-password: {{ .Values.keycloak.keycloak.password | b64enc | quote }}
+  keycloak-postgres-password: {{ .Values.global.keycloak.postgresPassword | b64enc | quote }}
   renku-realm.json: {{ include "renku.keycloak.realm" . | indent 4  | b64enc | quote }}
 {{- end }}
-  jupyterhub-password: {{ .Values.global.jupyterhub.postgresPassword | b64enc | quote }}
+  jupyterhub-postgres-password: {{ .Values.global.jupyterhub.postgresPassword | b64enc | quote }}
```

Comment:
Corresponding changes in charts/renku/templates/configmap.yaml and other templates are missing

Reply:
@leafty : yes, I was suspecting that, this is mid-flight state